e2e/viewHelper.js

```diff
@@ -181,3 +182,11 @@ export const switchToGoreliNetwork = async () => {
   await WalletView.isNetworkNameVisible(GOERLI);
   await NetworkEducationModal.tapGotItButton();
 };
+
+export const skipOnboarding = async () => {
+  const PASSWORD = '123123123';
+  await LoginView.isVisible();
+  await LoginView.enterPassword(PASSWORD);
```

Comment:
You can import the password from the validAccount object. 

```suggestion
  await LoginView.enterPassword(validAccount.password);
```

Reply:
I was trying to avoid disruption here because the password on the fixtures account is different from the one in `Accounts.js`.  One thing to use `validAccount.password` for this case again is if I change the password in the fixtures `KeyringController` to match the one in `validAccount.password`.